src/main.tsx

```diff
@@ -0,0 +1,9 @@
+import { createRoot } from "react-dom/client";
+import { StrictMode } from "react";
+import App from "./App.tsx";
+
+createRoot(document.getElementById('root')!).render(
```

Comment:
Rather than asserting here a clearer option would be to use cast typing e.g. 
("root") as HTMLElement

Reply:
Fixed. Also had to make an explicit null check for root container.